tools/ui-components/src/form-group/form-group.tsx

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+
+import React, { Component, useEffect } from 'react';
+
+import { FormGroupProps } from './types';
+
+const FormGroup = React.forwardRef<HTMLLabelElement, FormGroupProps>(
+  ({ children, className, validationState, controlId }): JSX.Element => {
```

Comment:
This component is a `div` so the type should be checking if it's `HTMLDivElement`.

I am wondering if it should be using `props` instead of children, to check for its component.

I.e I should test how it acts with `props`, instead of `ReactNode` type.

`UseEffect` requires the usage of ref, adding ref attribute in the component should have fixed it